m2e-maven-runtime/pom.xml

```diff
@@ -115,7 +68,7 @@
 							<_failok>true</_failok>
 							<_nouses>true</_nouses>
 							<_nodefaultversion>true</_nodefaultversion>
-							<_snapshot>${commitDate}</_snapshot>
+							<_snapshot>qualifier</_snapshot>
```

Comment:
so how does this happen at build-time?
The _snapshot is 1st set to qualifier in the 1st `mvn` command and then in the 2nd, the `qualifier` is replaced with the last git commit timestamp using the usual tycho version computation?

Reply:
Exactly. The 1st build creates a MANIFEST.MF that contains a header like `Bundle-Version: 1.18.1.qualifier`, which is then processed as usual by Tycho.